.gitignore

```diff
@@ -65,6 +65,4 @@ typings/
 
 # next stuff
 .next
-
-# npm package-lock.json
-package-lock.json
```

Comment:
Please restore this.

Reply:
Yes I can do this but at least can you briefly explain why you don't want it? Just curious.

Reply:
I think that the reason is to ensure that CI checks are done with the latest compatible version of the dependencies. Otherwise the library could break without noticing.

Reply:
Please read a detailed explanation in https://github.com/sindresorhus/ama/issues/479#issuecomment-310661514.